src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -237,6 +239,22 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
             WorkspaceService.ListAllWorkspaces)
       }
     } ~
+    path("admin" / "publishedWorkspaces") {
+      get {
+        requestContext => perRequest(requestContext,
+          WorkspaceService.props(workspaceServiceConstructor, userInfo),
+          WorkspaceService.ListPublishedWorkspaces)
+      }
+    } ~
+    path("admin" / "workspacesByAttribute") {
+      post {
+        entity(as[AttributeMap]) { attributeMap =>
```

Comment:
since we only handle a single key/value, this should take a single key/value, not a map. Also, this probably should be a Get not a post.